File: code/TemplateStudioForUWP/Templates/_comp/MT/OptionalLogin._VB/ViewModels/SettingsViewModel_postaction.vb
Namespace ViewModels
    Public Class SettingsViewModel
        Inherits ObservableObject

        Public Property IsBusy As Boolean
            Get
                Return _isBusy
            End Get
            Set(value As Boolean)
                [SetProperty](_isBusy, value)
'^^
'{[{
                LogInCommand.NotifyCanExecuteChanged()
                LogOutCommand.NotifyCanExecuteChanged()
'}]}
            End Set
        End Property
    End Class
End Namespace

File: code/TemplateStudioForUWP/Templates/Uwp/_comp/MT/Service.IdentityOptionalLogin/ViewModels/SettingsViewModel_postaction.cs
//{[{
using Microsoft.Toolkit.Mvvm.Input;
//}]}

namespace Param_RootNamespace.ViewModels
{
    public class SettingsViewModel : ObservableObject
    {
        public bool IsBusy
        {
            get => _isBusy;
            set
            {
                SetProperty(ref _isBusy, value);
//^^
//{[{
                LogInCommand.NotifyCanExecuteChanged();
                LogOutCommand.NotifyCanExecuteChanged();
//}]}
            }
        }
    }
}
